src/Language/Fixpoint/Smt/Theories.hs

```diff
@@ -515,6 +525,13 @@ interpSymbols =
                                  $ FFunc (bagSort $ FVar 0)
                                          (bagSort $ FVar 0)
     bagSubSort = FAbs 0 $ FFunc (bagSort $ FVar 0) $ FFunc (bagSort $ FVar 0) boolSort
+
+bv2i :: Config -> Int -> Raw
+bv2i cfg size = case solver cfg of
+  Cvc4 -> "bv2nat"
+  Cvc5 -> "bv2nat"
```

Comment:
Is this correct? Sounds like `bv2nat` has different semantics than `bv2int`

Reply:
https://github.com/Z3Prover/z3/issues/1252

Reply:
apparently they do the same thing... (per Nikolaj B.)

Reply:
`cvc` people say is not standard smtlib https://github.com/cvc5/cvc5/issues/11252, but I see it here https://smt-lib.org/theories-FixedSizeBitVectors.shtml

Reply:
Yes, its odd. Furthermore -- CVC seems to just _not support_ `bv2int` -- I get `Symbol 'bv2int' not declared as a variable` errors...? I guess you better only implicitly convert `nat`s into `bv`...

Reply:
Btw, https://github.com/Z3Prover/z3/pull/6419 seems to indicate that even in z3 `bv2int` and `bv2nat` do the same thing?

Reply:
Oh I get it now, `bv2int` and `bv2nat` are not part of the standard they are only defined to define the semantics. From the CVC5 issue it seems that is going to be standardized in smtlib 2.7 so maybe worth tracking in an issue. 

Also is bv2int/bv2nat in z3 the same as bv2nat in CVC?

Reply:
Ok I get it now.

* In Z3 `bv2int` and `bv2nat` are aliases. They both implement the semantics of the upcoming `bv2nat`, i.e., they interpret the bit vector as an unsigned integer
* In CVC, only `bv2nat` is implemented with the semantics of the upcoming `bv2nat`

In smtlib 2.7, `bv2int` will have a different meaning (bit vectors as signed integer), so the correct thing here is to use `bv2nat` both for z3 and cvc because it will be forward compatible.

Fixpoint (and users of fixpoint) should probably start using `*_to_nat` instead of `*_to_int`  to reserve `to_int` for the upcoming meaning in 2.7.